readme.md

```diff
@@ -1,5 +1,13 @@
 # Microsoft Azure IoT SDK for .NET
 
+## 2.0.0 clients release notice
+
+A [2.0.0 preview release](https://github.com/Azure/azure-iot-sdk-csharp/releases/tag/2022-10-14-v2preview) has been published for each package in this library. 
+With this release, all future features will be brought only to this new major version, so users are highly encouraged to migrate from the 1.X.X releases to 
+the new 2.0.0 releases.
```

Comment:
We might want to word this a bit differently, given that the v2 is currently in preview. 
How about we say something like - A 2.0.0 preview prelease has been published for each package in this library. These packages simplify usage and handling of our clients, and we highly encourage you to try out these packages and share your feedback. Once we promote these changes to `main` in the future, all future features will be brought only to this new major version.

Thoughts?

Reply:
Yeah, I like it!